packages/app/src/app/vscode/initializers.ts

```diff
@@ -106,7 +106,7 @@ export function setVimExtensionEnabled(vimEnabled: boolean) {
   } else {
     // Auto disable vim extension
     if (
-      localStorage.getItem('vs-global://extensionsIdentifiers/disabled') ==
+      localStorage.getItem('vs-global://extensionsIdentifiers/disabled') ===
```

Comment:
This has to be `==`, to also account for `null` values.

Reply:
Will make it an
```ts
[null, undefined].includes( /* ... */)
```
instead so we don't have to disable the linting

Reply:
I think the rule has an exception for `== null`, I'm not 100% sure though!

Reply:
But you're checking on `undefined` here so the rule errors out.

Doing both a check to `null` and `undefined` is the same as doing the
```ts
[null, undefined].includes(/* ... */)
```
Which is a bit cleaner than the double check I think, no? 🤔